packages/terra-collapse/tests/nightwatch/collapse-spec.js

```diff
@@ -0,0 +1,64 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+
+const screenshot = require('terra-toolkit').screenshot;
```

Comment:
You should also get @mmalaker 's  opinion as well. When I wrote tests for both controlled and uncontrolled form inputs, it confused him as the components were doing the same exact thing, and was hard to determine what all was needed to be tested.